### PR TITLE
Samsung Galaxy S, S II, S III, S6 and S6 Edge.

### PR DESCRIPTION
**Pull request by incognitojam**
I have added the device screen details for both the S6 and S6 Edge. Both devices have the same screen dimensions etc.

### CODE REVIEW

**Comment by incognitojam** [comment]
I've now added the S III, S II and S.

File: screens.json [hunk 1]
```diff
@@ -614,5 +614,33 @@
 		"w": 1080,
 		"h": 1920,
 		"d": 5.5
+	},
+	{
+		"name": "Samsung Galaxy S6, S6 Edge",
```
Comment (LeaVerou) [inline screens.json:619]: Are these two phones? If so, they should be separate entries.

**Comment by LeaVerou** [comment]
Thanks! See my inline comment.

File: screens.json (hunk 1 above)
Comment (incognitojam) [inline screens.json:619]: They are both the same phone in every way, apart from the external design on the screen on the S6 Edge. Their resolutions are the same, so I decided to put them under one entry like the iPhone 4 and 4S entry already shown in the table.

**Comment by LeaVerou** [comment]
Merged, thanks! The comment was for searchability but I guess that won't be affected much.

**Pull request merged**